### PR TITLE
fix: tab overflow on smaller viewport and tab scroll ux

### PR DESCRIPTION
**Pull request by G3root**
before:

Screen Shot 2023-08-26 at 07 24 30

after:

Screen Shot 2023-08-26 at 07 25 18

### CODE REVIEW

File: apps/web/src/app/(dashboard)/documents/page.tsx [hunk 1]
```diff
@@ -70,11 +70,14 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
...
-        <Tabs defaultValue={shouldDefaultToPending ? InternalDocumentStatus.PENDING : status}>
+      <div className="mt-8 flex flex-wrap gap-x-4 gap-y-6 ">
+        <Tabs
+          className="overflow-auto"
+          defaultValue={shouldDefaultToPending ? InternalDocumentStatus.PENDING : status}
+        >
           <TabsList>
             <TabsTrigger className="min-w-[60px]" value={InternalDocumentStatus.PENDING} asChild>
-              <Link href={getTabHref(InternalDocumentStatus.PENDING)}>
+              <Link href={getTabHref(InternalDocumentStatus.PENDING)} scroll={false}>
```
Comment (G3root) [inline apps/web/src/app/(dashboard)/documents/page.tsx:80]: to prevent scroll to top when a user changes the tab

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

File: apps/web/src/app/(dashboard)/documents/page.tsx [hunk 2]
```diff
@@ -70,11 +70,14 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
 
       <h1 className="mt-12 text-4xl font-semibold">Documents</h1>
 
-      <div className="mt-8 flex flex-wrap gap-x-4 gap-y-6">
-        <Tabs defaultValue={shouldDefaultToPending ? InternalDocumentStatus.PENDING : status}>
+      <div className="mt-8 flex flex-wrap gap-x-4 gap-y-6 ">
+        <Tabs
+          className="overflow-auto"
```
Comment (Mythie) [inline apps/web/src/app/(dashboard)/documents/page.tsx:75]: This should probably just be `overflow-x-auto` since we don’t want any y axis scrolling 😄

**Comment by G3root** [comment]
@Mythie updated the PR with the requested changes and signed the CLA

**Review by Mythie — APPROVED** [review]

**Comment by Mythie** [comment]
Thanks Nafees 🫶🏻

thanks-pug

**Pull request merged**